src/Cli/dotnet/commands/dotnet-package/search/PackageSearchCommandParser.cs

```diff
@@ -97,6 +97,13 @@ private static CliCommand ConstructCommand()
                 {
                     parseResult.ShowHelp();
                 }
+
+                if (exitCode != 0)
+                {
+                    return 1;
+                }
+
+                return 0;
```

Comment:
This _might_ look better?
```suggestion
                // Only return 1 or 0
                return exitCode == 0 ? 0 : 1;
```